### ISSUE TITLE
Config request: toggle for the "jump to launch rockets" ability

### ISSUE THREAD

**Issue by CryinMo** [Lizard-Of-Oz/Inventorio#150]
It would be great to be able to either toggle ON/OFF the usage of the jump button while flying to launch rockets because it makes it virtually impossible to ascend in water while wearing elytra (or swim up flowing water columns). It would actually be perfect if the jump action triggered the rockets while in the main inventory as is currently the case BUT if you placed the rockets into the offhand utility belt, then the rockets would only fire when actually held in the off-hand as per vanilla.

**Comment by Grim-Medic** [comment]
Yes Please. Not having control over my rockets make this entire mod un-usable. It's great otherwise and i really think it fixes the biggest problem in minecraft

**Comment by Lizard-Of-Oz** [comment]
You can re-bind the "use rockets" action to a throwaway button you never use. Not the pretties option, but an option still.

**Comment by 4P5** [comment]
Would it be possible to allow for `None` as a rebind? For other controls, you can press `Esc` to disable it.

**Comment by dranorter** [comment]
I'd like to second (or, third or fourth) some sort of disable option for this. Allowing a 'none' rebind is pretty logical, but I'd also like a simple toggle in the mod settings, for no other reason than that being the first place I looked. The auto-rockets can be a VERY frustrating feature for those of us really used to Minecraft's controls; I often just want to glide to get somewhere, and swimming up is a pretty important feature too! I also typically spend my mid-game very conscious of how many rockets I'm using up, and have plenty of accidental rocket firings without a whole extra way to do so.
